### PR TITLE
Add Abuse[.]ch trusted reporter lists from URLhaus and MalwareBazaar

### PR DESCRIPTION
**Pull request by hugh-sublime**
# Context

We now have a daily lambda that filters lists from Abuse[.]ch URLhaus and MalwareBazaar down to just the trusted providers (outlined in the display name). The newly added lists (available at `https://global-sublime-static-lists.s3.amazonaws.com/latest`):

* `abuse_ch_malwarebazaar_trusted_reporters.csv` (83.2 MB)
  * Filtered the MalwareBazaar CSV down to just the trusted reporters. This includes all the fields
* `abuse_ch_malwarebazaar_sha256_trusted_reporters.txt` (12.1 MB)
  * This is the `sha256_hash` column from `abuse_ch_malwarebazaar_trusted_reporters.csv`
* `abuse_ch_urlhaus_trusted_reporters.csv` (3.4 MB)
  * Filtered the URLhaus CSV down to just the trusted reporters. This includes all the fields
* `abuse_ch_urlhaus_urls_trusted_reporters.txt` (635.5 KB)
  * This is the `url` field from `abuse_ch_urlhaus_trusted_reporters.csv`
* `abuse_ch_urlhaus_domains_trusted_reporters.txt` (191.4 KB)
  * This is the `domain` part of the URLs from `abuse_ch_urlhaus_urls_trusted_reporters.txt`
  * Duplicates are removed

While we would be able to derive some of the lists from just the filtered CSV files, we probably want to avoid that since they are pretty large to download. We can work with the preprocessed lists for now and always change it to the larger CSVs for flexibility in the future.

In addition to the lists being written to `https://global-sublime-static-lists.s3.amazonaws.com/latest`, we also write the daily lists to a date folder as well. For example, `https://global-sublime-static-lists.s3.amazonaws.com/2023-05-18/`. We do this in case we want to "roll back" or pin to a previous list.

# Tests

I pointed my local instances to my fork's branch. Verified that the lists get imported and populated:

Screenshot 2023-05-18 at 10 01 40 AM
Screenshot 2023-05-18 at 10 01 48 AM
Screenshot 2023-05-18 at 10 01 59 AM
Screenshot 2023-05-18 at 10 02 16 AM

I also tested out using the full CSVs (see commit 3f14ee7e3feab25ed93f27a56dcadca75ebd2ff0) and filtering down based on a column.

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -127,6 +127,30 @@
       "headers": [
         "entry"
       ]
+    },
+    {
+      "url": "https://global-sublime-static-lists.s3.amazonaws.com/latest/abuse_ch_malwarebazaar_sha256_trusted_reporters.txt",
+      "identifier": "abuse_ch_malwarebazaar_sha256_trusted_reporters",
+      "display_name": "Abuse.ch: MalwareBazaar SHA256 hashes. Trusted reporters: pr0xylife, abuse_ch, cocaman, James_inthe_box, JAMESWT_MHT\"",
```
Comment (rw-access) [inline manifest.json:134]:
```suggestion
      "display_name": "Abuse.ch: MalwareBazaar SHA256 hashes. Trusted reporters: pr0xylife, abuse_ch, cocaman, James_inthe_box, JAMESWT_MHT",
```

**Review by rw-access — COMMENTED** [review]
awesome! as long as the `url` support is released (looks like it has), then I'm down!

**Review by rw-access — APPROVED** [review]
LGTM. As long as the rest of @sublime-security/detection is on, I'm down

**Comment by morriscode** [comment]
This is fantastic! Nice work, can't wait to use this!!

**Comment by jkamdjou** [comment]
❤️❤️ what do y'all think of adding a disclaimer to the descriptions of these, that these are unfiltered lists and you should not use them on their own without some filtering in MQL?

**Comment by hugh-sublime** [comment]
@jkamdjou What do you mean by unfiltered in this case?

**Comment by jkamdjou** [comment]
> @jkamdjou What do you mean by unfiltered in this case?

the raw entries submitted by any trusted reporter (accidents happen)

**Review by jkamdjou — COMMENTED** [review]
took a stab at a disclaimer. cc @hugh-sublime @rw-access @morriscode

File: manifest.json [hunk 2]
```diff
@@ -127,6 +127,30 @@
...
+      "identifier": "abuse_ch_urlhaus_urls_trusted_reporters",
+      "display_name": "Abuse.ch: URLhaus URLs. Trusted reporters: Cryptolaemus1, abuse_ch",
+      "headers": [
+        "entry"
+      ]
+    },
+    {
+      "url": "https://global-sublime-static-lists.s3.amazonaws.com/latest/abuse_ch_urlhaus_domains_trusted_reporters.txt",
+      "identifier": "abuse_ch_urlhaus_domains_trusted_reporters",
+      "display_name": "Abuse.ch: URLhaus domains derived from trusted reporter URLs. Trusted reporters: Cryptolaemus1, abuse_ch",
```
Comment (rw-access) [inline manifest.json:150]: > derived from trusted reporter URLs

confusing. They are derived from URL lists and filtered by trusted reporters. This sounds like "trusted reporter URLs" is a thing

Comment (jkamdjou) [reply manifest.json:150]: how about `URLs derived from $abuse_ch_urlhaus_urls_trusted_reporters`

Comment (hugh-sublime) [reply manifest.json:150]: Minor tweak to: `Domains derived from $abuse_ch_urlhaus_urls_trusted_reporters`

**Comment by hugh-sublime** [comment]
I also don't have write access to this repo so either @rw-access or @jkamdjou will have to do it if it looks good!

**Review by jkamdjou — APPROVED** [review]

**Pull request merged**